config/schemas/config.yaml

```diff
@@ -127,6 +366,712 @@ properties:
+            type: object
+        type: object
+      opensearch:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+          plugins:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+              ports:
+                $ref: '#/$defs/portlist'
+                default:
+                  - 443
+            type: object
+        type: object
+      rcloneSync:
+        additionalProperties: false
+        properties:
+          destinationObjectStorageS3:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+              ports:
+                $ref: '#/$defs/portlist'
+                default:
+                  - 443
+            type: object
+          destinationObjectStorageSwift:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+              ports:
+                $ref: '#/$defs/portlist'
+                default:
+                  - 5000
+            type: object
+          enabled:
+            default: true
+            type: boolean
+          secondaryUrl:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+              ports:
+                $ref: '#/$defs/portlist'
+                default:
+                  - 443
+            type: object
+        type: object
+      rookCeph:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: false
+            type: boolean
+        type: object
+      s3Exporter:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      thanos:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      velero:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      enabled:
+        type: boolean
+      enforcement:
+        type: string
+        $comment: TODO enforcement enum?
+      rclone:
+        type: object
+        properties:
+          enabled:
+            type: boolean
+    type: object
+  nodeLocalDns:
+    additionalProperties: false
+    properties:
+      customConfig:
+        description: https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/#coredns-configmap-options
+        examples:
+          - |-
+            example.com:53 {
+              errors
+              cache 30
+              reload
+              loop
+              forward . 127.0.0.1:9005
+            }
+        type:
+          - string
+          - "null"
```

Comment:
Did you also try to update the [config](https://github.com/elastisys/compliantkubernetes-apps/blob/v0.37.0/config/config/common-config.yaml#L1090) to an empty string as well instead of adding `null` here as suggested by @aarnq ?

Reply:
Yes, in fe957df9224ea7381bf2be96f9255ece26bef59b